### PR TITLE
feature: add cooldown command

### PR DESCRIPTION
**Pull request by taka-oyama**

### CODE REVIEW

**Comment by taka-oyama** [comment]
Need to merge https://github.com/colopl/laravel-spanner/pull/53 first.

File: tests/Colopl/Spanner/Console/TestCase.php [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+namespace Colopl\Spanner\Tests\Console;
+
+use Colopl\Spanner\Tests\TestCase as BaseTestCase;
+use Illuminate\Testing\PendingCommand;
+
+class TestCase extends BaseTestCase
+{
+    /**
+     * @param $command
```
Comment (tyabu12) [inline tests/Colopl/Spanner/Console/TestCase.php:26]:
```suggestion
     * @param string $command
```

File: tests/Colopl/Spanner/Console/TestCase.php [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+     * @param $parameters
```
Comment (tyabu12) [inline tests/Colopl/Spanner/Console/TestCase.php:27]:
```suggestion
     * @param array $parameters
```

**Review by halnique — APPROVED** [review]
LGTM

**Pull request merged**